<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>question_form</class>
 <widget class="QWidget" name="question_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>96</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>96</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>480</width>
    <height>360</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Question</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resourses.qrc">
    <normaloff>:/icon_res/excel.svg</normaloff>:/icon_res/excel.svg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="text_label">
     <property name="text">
      <string>The text of question!</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontal_widget">
     <item>
      <widget class="QPushButton" name="button_no">
       <property name="text">
        <string>NO</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_ok">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resourses.qrc"/>
 </resources>
 <connections/>
</ui>
